phpBB/docs/events.md

```diff
@@ -1052,6 +1052,14 @@ posting_poll_body_options_after
 * Since: 3.1.4-RC1
 * Purpose: Add content after the poll options on creating a poll
 
+posting_topic_title_prepend
```

Comment:
To be honest, we never really decided upon using append/prepend or after/before. For non-native English speakers the latter one (after/before) will be easier to understand.